### PR TITLE
ADAPTOPS-125: One col no image localist styling

### PR DESCRIPTION
**Pull request by Lyafel**
# NOT READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- TL;DR - what's this PR for?

# Review By (Date)
- When does this need to be reviewed by?

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to...
3. Verify...

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Comment by yvonnetangsu** [comment]
@Lyafel I'm out sick today, but if you could point me to a preview link so I can review tomorrow, that'd be awesome. thanks!

**Review by yvonnetangsu — CHANGES_REQUESTED** [review]
@Lyafel Thanks for this PR. It's looking great - I only have a few minor requests :) 
1. Could the arrow icon after the link be larger (see similar treatment on the site)? Currently it's pretty small
Screenshot 2023-05-09 at 12 00 18 PM

2. Could we add a max-width: 75ch to the title link?
3. Could we pull up the superheading (class/seminar below) a bit so it vertically aligns with the month?
Screenshot 2023-05-09 at 12 03 00 PM
4. Could we top align these icons with the text next to them? (currently center aligned) These text can have a line height of 1.2


Screenshot 2023-05-09 at 9 35 05 AM

Thanks!

File: src/styles/localist.css [hunk 1]
```diff
@@ -80,3 +80,65 @@
...
+}
+
+.su-event-widget--1col-no-images .event-list .event .event-title {
+  padding: 0;
+  display: inline;
+}
+
+.su-event-widget--1col-no-images .event-list .event .event-title__link {
+  margin-bottom: 0.43em;
+  max-width: 75ch;
```
Comment (yvonnetangsu) [inline src/styles/localist.css:126]: I didn't realize the font size was 10px in this container. Turns out 75ch is a bit small here. Let's change this to 80rem instead @Lyafel 😄

Comment (Lyafel) [reply src/styles/localist.css]: lol I thought that looked funny, don't know why I didn't ask instead of pushing :D

**Comment by yvonnetangsu** [comment]
Thanks @Lyafel ! I like it :D One final ask - I meant to ask for the small icons to line up more like:
Screenshot 2023-05-10 at 11 48 04 AM

So ie, the first line of text would vertically center aligned with the icon - could we do that? 😄

**Comment by yvonnetangsu** [comment]
@Lyafel Thanks for the update on the icon - that looks great. The client would like the heading color and underline color (normal and on hover/focus) to match these event cards on the site
Screenshot 2023-05-11 at 12 07 31 PM

https://alumni.stanford.edu/ (under the Upcoming Events section)

if you could match those, that'd be awesome - thanks!

**Comment by Lyafel** [comment]
@yvonnetangsu Done, not sure why the preview still shows to be failed, I cleared the cache and reran it, https://app.netlify.com/sites/stanford-alumni/deploys/645eae76f6102444daa1d54d looks fine to me

**Comment by yvonnetangsu** [comment]
> @yvonnetangsu Done, not sure why the preview still shows to be failed, I cleared the cache and reran it, https://app.netlify.com/sites/stanford-alumni/deploys/645eae76f6102444daa1d54d looks fine to me

Yeah somehow the status doesn't seem to update with the successful preview. No worries - I'm making a few more tweaks myself then we can send this to the client for another review.

**Review by yvonnetangsu — APPROVED** [review]
Client has reviewed and this is GTG. @sherakama We'll cut a new release today later and get this into PROD. I believe we can merge the code first, then they can change the widget on the Storyblok side after 🤔

**Review by yvonnetangsu — APPROVED** [review]
Client has reviewed and this is GTG. @sherakama We'll cut a new release today later and get this into PROD. I believe we can merge the code first, then they can change the widget on the Storyblok side after 🤔

**Review by yvonnetangsu — APPROVED** [review]
Client has reviewed and this is GTG. @sherakama We'll cut a new release today later and get this into PROD. I believe we can merge the code first, then they can change the widget on the Storyblok side after 🤔

**Comment by sherakama** [comment]
You can lock the deployment on the Netlify site to the current build if you need to deploy code and then make Storyblok changes. That should give you time to make any changes without disruption to the live site.

**Comment by yvonnetangsu** [comment]
@sherakama Thanks for the tip! 🎉

**Pull request merged**